schemas/appeal-document.schema.json

```diff
@@ -0,0 +1,226 @@
+    },
+    "lastModified": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "The last update date for the document"
+    },
+    "caseType": {
+      "type": ["string", "null"],
+      "enum": [
+        "C",
+        "D",
+        "F",
+        "G",
+        "H",
+        "L",
+        "Q",
+        "S",
+        "V",
+        "W",
+        "X",
+        "Y",
+        "Z",
+        null
+      ],
+      "description": "The internal code for an appeal type, e.g. Householder (D)"
+    },
+    "redactedStatus": {
+      "type": ["string", "null"],
+      "enum": ["not_redacted", "redacted", "no_redaction_required", null],
+      "description": "Indicates the redaction status for the document"
+    },
+    "publishedStatus": {
+      "type": ["boolean"],
+      "description": "Indicates if a document can be shared with third-parties",
+      "$comment": "Consumers should expose documents that are published, and have a suitable redaction status"
+    },
+    "documentType": {
+      "type": ["string", "null"],
+      "enum": [
+        "appellantCaseCorrespondence",
+        "appellantCaseWithdrawalLetter",
+        "appellantCostsApplication",
+        "appellantCostsCorrespondence",
+        "appellantCostsWithdrawal",
+        "appellantStatement",
+        "applicationDecisionLetter",
+        "changedDescription",
+        "originalApplicationForm",
+        "whoNotified",
+        "conservationMap",
+        "lettersNeighbours",
+        "lpaCaseCorrespondence",
+        "lpaCostsApplication",
+        "lpaCostsCorrespondence",
+        "lpaCostsWithdrawal",
+        "lpaStatementDocuments",
+        "otherPartyRepresentations",
+        "planningOfficerReport",
+        "pressAdvert",
+        "siteNotice",
+        "costsDecisionLetter",
+        "caseDecisionLetter",
+        "crossTeamCorrespondence",
+        "inspectorCorrespondence",
+        null
+      ],
+      "description": "The type of document, used for exchange, migrations and consumption from the appeal back-office system"
+    },
+    "sourceSystem": {
+      "type": ["string", "null"],
+      "enum": ["back-office-appeals", "horizon", "acp", "sharepoint", null],
+      "description": "The system mastering the metadata for the current document"
+    },
+    "origin": {
+      "type": ["string", "null"],
+      "enum": ["pins", "citizen", "lpa", "ogd", null],
+      "description": "Indicates where the documents originates from"
+    },
+    "owner": {
+      "type": ["string", "null"],
+      "description": "Owner of the current document"
+    },
+    "author": {
+      "type": ["string", "null"],
+      "description": "Name of person who authored document"
+    },
+    "description": {
+      "type": ["string", "null"],
+      "description": "A custom description for the document"
+    },
+    "caseStage": {
```

Comment:
Unsure if we need this - probably the `origin` field could serve the same purpose